spanner/tests/system/test_system.py

```diff
@@ -973,6 +973,90 @@ def test_read_w_ranges(self):
         expected = all_data_rows[START+1:END+1]
         self._check_row_data(rows, expected)
 
+    def test_read_with_range_keys_index_closed_closed(self):
+        row_count = 10
+        columns = self.COLUMNS[1], self.COLUMNS[2]
+        session, committed = self._set_up_table(row_count)
+        self.to_delete.append(session)
+        expected = [[row[1], row[2]] for row in self._row_data(row_count)]
```

Comment:
`expected` is not a good name for this variable, as it no longer matches what we expect to be returned from the `read` call.  `data` would maybe be better?  Applies to all four testcases.